src/ui/FacetRange/FacetRange.ts

```diff
@@ -178,16 +164,78 @@ export class FacetRange extends Facet implements IComponentBindings {
 
   protected processNewGroupByResults(groupByResults: IGroupByResult) {
     if (groupByResults != null && this.options.ranges == null) {
-      groupByResults.values.sort((valueA, valueB) => {
-        const startEndA = valueA.value.split('..');
-        const startEndB = valueB.value.split('..');
-        if (this.options.dateField) {
-          return Date.parse(startEndA[0]) - Date.parse(startEndB[0]);
-        }
-        return Number(startEndA[0]) - Number(startEndB[0]);
-      });
+      groupByResults.values.sort((valueA, valueB) => this.sortRangeGroupByResults(valueA, valueB));
+      super.processNewGroupByResults(groupByResults);
+    }
+  }
+
+  private sortRangeGroupByResults(valueA: IGroupByValue, valueB: IGroupByValue) {
+    const startEndA = this.extractStartAndEndValue(valueA);
+    const startEndB = this.extractStartAndEndValue(valueB);
+    let firstValue: string;
+    let secondValue: string;
+
+    if (!startEndA && !startEndB) {
```

Comment:
That means that if exactly one of them has been extracted, we will try to use `.start` on both which will fail.